Userland/Libraries/LibC/getsubopt.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/ScopeGuard.h>
+#include <AK/StringView.h>
+#include <stdlib.h>
+#include <string.h>
+#include <AK/Format.h>
+#include <unistd.h>
+
+int getsubopt(char** option_array, char* const* tokens, char** option_value)
```

Comment:
I should probably put a label indicating this is a libc POSIX defined function:
https://pubs.opengroup.org/onlinepubs/9699919799/functions/getsubopt.html